Userland/Applications/SoundPlayer/SoundPlayerWidgetAdvancedView.cpp

```diff
@@ -180,6 +180,10 @@ void SoundPlayerWidgetAdvancedView::loop_mode_changed(Player::LoopMode)
 {
 }
 
+void SoundPlayerWidgetAdvancedView::mute_changed(bool)
+{
+}
```

Comment:
Hm... what's the purpose of this?

Reply:
It's because we need to update the volume widget to display if it's muted or not.

We could disable it (as the audio applet currently does) but I'd like to add a new widget that looks like this: 
![image](https://user-images.githubusercontent.com/26639800/144932461-b4c03807-2d5a-4428-a406-07132c941a59.png)

In the meantime, I moved the Fixme to this function 🎵